project/MicroServiceBuild.scala

```diff
@@ -26,19 +26,21 @@ private object AppDependencies {
   import play.core.PlayVersion
   import play.sbt.PlayImport._
 
-  private val domainVersion = "6.2.0-play-28"
-  private val scalaTestVersion = "3.0.9"
-  private val scalaTestPlusVersion = "5.0.0"
+  private val domainVersion = "8.0.0-play-28"
+  private val scalaTestVersion = "3.2.11"
+  private val scalaTestPlusVersion = "5.1.0"
   private val pegdownVersion = "1.6.0"
-  private val mockitoVersion = "3.7.0"
+  private val mockitoVersion = "4.4.0"
 
   private val scalaJVersion = "2.4.2"
 
   private val reactiveMongoVersion = "8.0.0-play-28"
 
-  private val wiremockVersion = "2.21.0"
+  private val wiremockVersion = "2.21.0" //2.32.0
```

Comment:
Did you try using `2.32.0`?

Reply:
You can add `"com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.2" % scope` to make it work.